packages/assemble-lite/helper/partials-helper.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
why does every file has 2 empty lines at the top ?

Reply:
i think it's because eslint removed all 'use-strict' lines

Reply:
fixed